gtfs-netex-test/netexio/dbaccess.py

```diff
@@ -585,8 +585,9 @@ def _prepare2(generator2, objectname):
     print('\n')
 
 def copy_table(db_read: Database, db_write: Database, classes: list, clean=False):
-    if db_read.read_only:
-        db_write.con.execute(f"ATTACH IF NOT EXISTS '{db_read.database_file}' AS db_read (READ_ONLY);")
+    if False and db_read.read_only:
```

Comment:
Please make this conditional based on the 'defauts' you have introduced. If it **is** a windows specific problem replace False with: `os.name != 'nt'`